packages/core/src/common/utils.ts

```diff
@@ -5,6 +5,14 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
+// only accessible within this file, so use `Utils.isNodeEnv` from the outside.
+declare var process: { env: any };
+
+/** Returns whether `process.env.NODE_ENV` exists and equals `env`. */
+export function isNodeEnv(env: string) {
```

Comment:
consider limiting this string type to just `"development" | "production" | "test"`